apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/style-tour.scss

```diff
@@ -152,16 +152,19 @@ $welcome-tour-button-background-color: #32373c; // former $dark-gray-700. TODO:
 		opacity: 0;
 		transition: opacity 200ms;
 	}
-}
 
-.wpcom-editor-welcome-tour-frame:hover .welcome-tour-card__overlay-controls .components-button,
-.welcome-tour-card__overlay-controls-visible .components-button {
-	opacity: 0.7;
-}
+	.wpcom-editor-welcome-tour-frame:hover &,
+	.wpcom-editor-welcome-tour-frame:focus-within &,
+	&-controls-visible {
```

Comment:
NIT. I generally avoid concatenating class names this way as it makes it harder to search for e.g. a plain search for `welcome-tour-card__overlay-controls-visible` will not bring this up in the results. 🤷